config/notifiers.go

```diff
@@ -101,14 +101,8 @@ var (
 		NotifierConfig: NotifierConfig{
 			VSendResolved: true,
 		},
-		Message:   `{{ template "wechat.default.message" . }}`,
-		APIURL:    `{{ template "wechat.default.api_url" . }}`,
-		APISecret: `{{ template "wechat.default.api_secret" . }}`,
-		ToUser:    `{{ template "wechat.default.to_user" . }}`,
-		ToParty:   `{{ template "wechat.default.to_party" . }}`,
-		ToTag:     `{{ template "wechat.default.to_tag" . }}`,
-		AgentID:   `{{ template "wechat.default.agent_id" . }}`,
```

Comment:
Why were all these fields removed?

Reply:
I think these fields should be a determined value.